lib/modules/datasource/jsr/util.ts

```diff
@@ -0,0 +1,54 @@
+// Ported from jsr:
+// https://github.com/jsr-io/jsr/blob/b8d753f4ed96f032bc494e8809065cfe8df5c641/frontend/utils/ids.ts
+// Copyright 2024 the JSR authors. All rights reserved. MIT license.
+
+import is from '@sindresorhus/is';
+
+export function extractJsrPackageName(packageName: string): {
+  scope: string;
+  name: string;
+} | null {
+  // trim first @
+  const validatePackageName = packageName.replace(/^@/, '');
```

Comment:
```suggestion
  const validatePackageName = packageName.replace(regEx(/^@/), '');
```
use our regex helper